README.md

```diff
@@ -15,32 +15,36 @@ This is a fuzzer to test the correctness of various [top-level await](https://gi
 
 Variant: simple
 
-* Custom module registry algorithm: ✅ Same (100% same)
 * Webpack 5.75.0: ✅ Same (100% same)
+* Rspack 0.3.8: ✅ Same (100% same)
+* Custom module registry algorithm: 🚫 Different (100% same)
```

Comment:
@evanw It's worth noting that there are some slight deviations from other Bundlers. I'm not sure if these changes should be ignored here, but it will affect the execution of the automation script.